plugins/mcreator-localization/help/ru_RU/item/when_right_clicked_block.md

```diff
@@ -1,4 +1,9 @@
 ﻿Вызывает процедуру при щелчке ПКМ игрока по блоку с этим предметом в руке.
 
-Результат выполнения процедуры должен быть _SUCCESS/CONSUME_ при успешном взаимодействии, _FAIL_ - в случае неудачного
-взаимодействия, _PASS_ - при его отсутствии. Если процедура ничего не возвращает, то результат будет _PASS_.
\ No newline at end of file
+SUCCESS/CONSUME при успешном взаимодействии,

```

Comment:
Seems you forgot something:
```suggestion
Результат выполнения процедуры должен быть SUCCESS/CONSUME при успешном взаимодействии,
```

Reply:
I'm not an expert in Russian, but is there no better way to say that it returns the value as a result of an action? From what I can tell, this reads more like "the result of the action should be..." rather than "the procedure should return as a result of..."

Reply:
:thinking: